### ISSUE TITLE
'mobx' does not contain an export named '$mobx'

### ISSUE THREAD

**Issue by pavsidhu** [mobxjs/mobx-react#494]
After installing mobx and mobx-react, I receive the following error:

```
./node_modules/mobx-react/index.module.js
599:24-29 'mobx' does not contain an export named '$mobx'.
```

I've used this library in the past and have had no issues. I'm unsure if this is an issue on my end or not. Let me know if there's anything I can do to help.

**Comment by garkin** [comment]
Getting the same after updating to `mobx-react@5.2.0`.
```json
"dependencies": {
    "mobx": "^4.3.0",
    "mobx-react": "^5.2.0",
    "webpack": "^4.10.2",
    "webpack-cli": "^3.0.1",
    "webpack-dev-server": "^3.1.4",
}
```

**Comment by garkin** [comment]
Loks related https://github.com/mobxjs/mobx-react/commit/18fc79730c581b4b4348cc6ed4b2f271aeddea91
Dudes, you made release minor version increment dependant on `"mobx": "5.0.0-beta.5"`. Not cool.

**Comment by mweststrate** [comment]
Upgrading mobx to 5.3.1 or downgrading mobx-react to 5.1.0 should solve the issue

**Comment by mweststrate** [comment]
Yeah, this was supposed to use dynamic feature detection, to figure if we
are dealing with MobX 4 or 5, but since the static analysis of webpack
can't handle that this breaks if you are bundling with webpack. Just
released mobx 4.3.1 to make it forward compatible. Doesn't solve the issue
100% (still incompatible with 4.3.0), but I am not sure if there is a real
use case where mobx-react is upgraded, but mobx not? Alternatively I can
retract mobx-react@5.2.0 and re-release it as 6.0.0, but that might
existing builds as well.

Op ma 4 jun. 2018 om 18:39 schreef garkin <notifications@github.com>:
